.github/workflows/python-ci-tests.yml

```
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3.9, '3.10']
+        python-version: [3.8, 3.9, '3.10', '3.13']
```

```suggestion
        python-version: ['3.13']
```

we could just test for a single version and save on runner execution times